diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 09d831da880738f9f92318a1b8944232f773caf7..6348f37fea53ab66d267522c5ae0a6093229b429 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -392,6 +392,7 @@ exports.networks = [
     { chainId: 656476n, shortName: 'open-campus-codex' },
     { chainId: 660279n, shortName: 'xai' },
     { chainId: 668668n, shortName: 'cnw' },
+    { chainId: 688688n, shortName: 'pharos-testnet' },
     { chainId: 695569n, shortName: 'pyrope' },
     { chainId: 713715n, shortName: 'sei-devnet' },
     { chainId: 743111n, shortName: 'hemi-sep' },
